index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=EB+Garamond:ital,wght@0,400;0,600;1,400;1,700&family=Inter:wght@400;700&family=Shantell+Sans:ital,wght@0,300;0,400;0,500;0,700;1,500;1,700&family=Tangerine&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital@1&display=swap" rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@3.2.0/fonts/remixicon.css" rel="stylesheet">
+    <link rel="stylesheet"
+    href="https://unpkg.com/boxicons@latest/css/boxicons.min.css">
+    <title>Bett Enock Portfolio</title>
+</head>
+<body>
+    <header class="icons">
+        <nav class="nav-container">
+            <div class="mylogo">
+                <a href="#"><img src="/Assets/images/cvimage2.JPG" alt="logo"></a>
+            </div>
+            
+            <ul class="nav-list">
+                <li> <a href="#" class="active">About</a></li>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+            <div class="menu_icons">
+                <a href="#"><img class="harmburgericon" src="/Assets/images/Harmburger Menu.svg" alt="Closeicon"></a>
+                <a href="#"><img class="closeicon" src="/Assets/images/Closeicon.svg" alt="Closeicon"></a>
+            </div>
+        </nav>
```

Comment:
Nice job fixing the broken links as requested by the previous reviewer👍. Currently, your toolbar has a lot of spacings (top and bottom) making it look a bit different from the given design😉. I think you can make the image logo smaller so as to retain the height of the toolbar as shown in the design😉.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="350" alt="image" src="https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/64643995/2a58c5e1-1810-473a-9cc3-a97724b679d1">|<img width="350" alt="image" src="https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/64643995/4a59f86d-b68d-4aaf-8088-293026b1059b">|